suite-common/wallet-core/src/send/sendFormSelectors.ts

```diff
@@ -50,13 +50,15 @@ export const selectSendFormReviewButtonRequestsCount = (
 
     const isCardano = networkType === 'cardano';
     const isEthereum = networkType === 'ethereum';
+    const isStellar = networkType === 'stellar';
```

Comment:
Hi @tomasklim, finished. 

P.S. This new change is due to the relocation of the relevant code, which is consistent with the previous changes.